components/Form/AddFlightForm.js

```diff
@@ -0,0 +1,322 @@
+import {
+  StyledForm,
+  Card,
+  StyledAddButton,
+  StyledHeading,
+  InputLabel,
+  InputBox,
+  InputField,
+  StyledInfoText,
+} from "./AddFlightForm.styled";
+import { useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
+import { flights } from "@/data/septemberFlights";
+import { useRouter } from "next/router";
+
+export default function AddFlightForm() {
+  const [flightsOfInterest, setFlightsOfInterest] = useLocalStorageState(
+    "flightsInfo",
+    { defaultValue: flights }
+  );
+
+  const router = useRouter();
+
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const flightData = Object.fromEntries(formData);
+
+    setFlightsOfInterest([...flightsOfInterest, flightData]);
+    router.push("/flight_list");
+    event.target.reset();
+    event.target.flight_iata.focus();
+  }
+
+  return (
+    <>
+      <StyledForm onSubmit={handleSubmit}>
+        <Card>
+          <StyledHeading>Schedule your next flight</StyledHeading>
+          <InputBox>
+            <InputLabel htmlFor="flight_iata" isFocused={isFocused}>
+              Flight #
+            </InputLabel>
+            <InputField
+              id="flight_iata"
+              type="text"
+              name="flight_iata"
+              required
+              maxLength="7"
+              pattern="^[A-Za-z]{2,3}\d{1,4}[A-Za-z]?$"
+              onFocus={handleFocus}
+              onBlur={(event) => {
+                handleBlur(event);
```

Comment:
The `event` object isn't needed for `handleBlur`.
```suggestion
                handleBlur();
```